frontend/src/bookings/calendar.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <div class="title inline-title">{{ title }}</div>
+    <keep-alive>
```

Comment:
What does keep-alive do?

Reply:
keep-alive keeps the instance of the calendar component alive between re-renders which is necessary because it is controlled via refs and the refs are created and destroyed between renders so it throws errors if an attempt to access the ref is made in between renders and the calendar component re-renders itself a lot.